connectors/source_asana/models/fivetran_converter/project.sql

```diff
@@ -0,0 +1,26 @@
+{% if target.type == "snowflake" %}
+
+{% elif target.type == "bigquery" %}
+    WITH tmp AS (
+        SELECT
+            t.gid as id,
+            t.archived,
+            t.created_at,
+            t.current_status,
+            t.due_date,
+            t.modified_at,
+            t.name,
+            JSON_QUERY(t.owner, '$.gid') as owner_id,
+            t.public,
+            JSON_QUERY(t.team, '$.gid') as team_id,
+            JSON_QUERY(t.workspace, '$.gid') as workspace_id,
+            t.notes
+        FROM
+            {{ source('source_asana', 'projects') }} t
+    )
```

Comment:
Nice start @pangeran-bottor you can use Chatgpt to translate to snowflake and postgres.